components/esp_eth/src/mac/esp_eth_mac_esp.c

```diff
@@ -364,6 +364,14 @@ esp_err_t emac_esp_custom_ioctl(esp_eth_mac_t *mac, int cmd, void *data)
                             "failed to set PTP target time");
         break;
     }
+    case ETH_MAC_ESP_CMD_ENABLE_TS4ALL: {
+	if (data == NULL) {
+	    return ESP_ERR_INVALID_ARG;
+	}
```

Comment:
```suggestion
    ESP_RETURN_ON_FALSE(data, ESP_ERR_INVALID_ARG, TAG, "PTP enable TS for all invalid argument, cant' be NULL");
```
Please follow the pattern.